src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -213,7 +213,7 @@ static AnimationInfoS sAnimations[] = {
     { &object_hakugin_demo_Anim_003378, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
 };
 
-EnGoStruct* func_80A10FD0(EnGoStruct ptr[], Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5, s32 arg6) {
+EnGoStruct* func_80A10FD0(EnGoStruct* ptr, Vec3f arg1, Vec3f arg2, Vec3f arg3, f32 arg4, f32 arg5, s32 arg6) {
```

Comment:
Since the functions on this file actually expect an array, and an array is passed, I think the function should use the `[]` syntax instead of the `*` one

Reply:
There are only 11 instances of this in the codebase at the moment, I think it confuses more than it helps, so I'd prefer not.